src/modules/roc_netio/target_uv/roc_netio/handle.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2019 Roc authors
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+//! @file roc_netio/target_uv/roc_netio/handle.h
+//! @brief Handle.
+
+#ifndef ROC_NETIO_HANDLE_H_
+#define ROC_NETIO_HANDLE_H_
+
+#include "roc_packet/address.h"
+
+namespace roc {
+namespace netio {
+
+//! Network handle.
+struct Handle {
+    //! Construct empty handle.
+    Handle()
+        : data(NULL)
+        , fn(NULL) {
+    }
+
+    //! Data to call callback with.
+    void* data;
+
+    //! Handle callback.
+    void (*fn)(void*, packet::Address&);
```

Comment:
Using address as a sender/receiver identifier is probably not very good because:

* we may want to allow sender and receiver to share the same UDP address
* we may have UDP and TCP sender/receiver with the same IP and port

I think it would be better to use `void *` as an identifier.